### DISCUSSION TITLE
Cannot play mp3 files on github pages page :(

### DISCUSSION THREAD

**Discussion by MaxSchreff** [community/community#30145] [Other Feature Feedback, Questions, & Ideas]
Hey,
I wanted to publish my virtual piano project with git hub pages. 
It looks like this:
![image](https://user-images.githubusercontent.com/111394953/185703276-be195ee6-d8f7-474b-9f1e-bdfbc2a9f6f4.png)
When you press the keys, the sound of the keys will be played. For that I am using a JS event handler
![image](https://user-images.githubusercontent.com/111394953/185703405-355b033a-9030-4ef4-a961-2e09b25dedfa.png)
It works on WebStorm but not on the github pages page :(
![image](https://user-images.githubusercontent.com/111394953/185703540-d6db3339-3553-45cb-9782-0ac8ebe14f67.png)
Does someone know how I can play the mp3 files over github?

**Comment by airtower-luna** [comment]
It's hard to say for sure without a link to the repository, but most likely there's some kind of mistake in the path you use to access the file.

**Comment by jfullstackdev** [comment]
it's playing 

![image](https://user-images.githubusercontent.com/47092464/186656632-09a2e2ea-e0c1-4570-8314-5b924b43c351.png)

but since the condition in the JS is `keydown`, you need to press the key and it will not work through the mouse, another issue here that may arise is keyboard mapping on different OS

![image](https://user-images.githubusercontent.com/47092464/186656847-88bb9464-c0a1-4843-986e-b729d0065a9c.png)

**Comment by radar-blip** [comment]
You got Z and Y mixed up.
There's a Z on keyboard, but no sound.
No Y on keyboard, but Y makes sound. 🎹 🎵 

keyY in code, no keyZ
